src/modules/0.0.16/aws_iam/index.ts

```diff
@@ -138,7 +138,23 @@ export const AwsIamModule: Module2 = new Module2({
       // EC2 role instance profile ARN example - arn:aws:iam::257682470237:instance-profile/test-role
       return  arn.split('/').pop();
     },
-    rolePolicyComparison: (a: any, b: any) => isEqual(a, b),
+    // rolePolicyComparison: (a: any, b: any) => isEqual(a, b),
+    rolePolicyComparison: (a: any, b: any) => {
+      if (Object.is(a, b)) return true;
+      if (Object.is(a, null) || Object.is(b, null) || !Object.is(typeof a, 'object') || !Object.is(typeof b, 'object')) return false;
+      const aKeys = Object.keys(a);
+      const bKeys = Object.keys(b);
+      if (!Object.is(aKeys.length, bKeys.length)) return false;
+      if (Array.isArray(a) && Array.isArray(b)) {
+        return a.every(ai => !!b.find(bj => AwsIamModule.utils.rolePolicyComparison(ai, bj)))
```

Comment:
Vulnerable to duplicate records, like this:

![Screenshot from 2022-08-05 10-22-33](https://user-images.githubusercontent.com/765551/183109781-de1017be-e15d-45b1-a6c9-e109fcdb25e4.png)

Working with sorted arrays will help, though:

![Screenshot from 2022-08-05 10-24-13](https://user-images.githubusercontent.com/765551/183109854-32282926-04f3-417b-9f67-abdc619211d3.png)


```suggestion
        const aSorted = [...a].sort();
        const bSorted = [...b].sort();
        for (let i = 0; i < aSorted.length; i++) {
          if (!AwsIamModule.utils.rolePolicyComparison(a[i], b[i])) return false;
        }
        return true;
```